resources/windows/wix/project.wxs.erb

```diff
@@ -55,16 +55,22 @@
     <!-- INSTALLDIR -->
     <CustomAction Id="SaveCmdLineInstallDir" Property="CMDLINE_INSTALLDIR" Value="[INSTALLDIR]" Execute="firstSequence" />
     <CustomAction Id="SetFromCmdLineInstallDir" Property="INSTALLDIR" Value="[CMDLINE_INSTALLDIR]" Execute="firstSequence" />
+    <%= File.read("resources/windows/CustomActions") if File.exist?("resources/windows/CustomActions") -%>
```

Comment:
The erb fragments presented by this approach were what I was hoping for as IMHO the most flexible approach and had discussed with @jamesp.